OsmAnd/src/net/osmand/plus/poi/PoiFiltersHelper.java

```diff
@@ -486,7 +460,7 @@ public boolean editPoiFilter(PoiUIFilter filter) {
 
 	public void addSelectedPoiFilter(PoiUIFilter filter) {
 		if (filter.isTopWikiFilter()) {
```

Comment:
Wiki is here?

Reply:
1. Remove wikipedia top filter in poi_types.xml
2. Add plugin method - Register custom POI Filter. In wikipedia plugin add Top Wiki Filter